### ISSUE TITLE
Example of how to send data

### ISSUE THREAD

**Issue by reiven** [klynch/python-logstash-handler#1]
Hello!

First of all, thanks for creating this lib, i cannot find any lib for sending data to logstash which uses ssl.
Second, i dont know if i'm doing something wrong, i just sending some string to logger object do not work at all
Do i need to send information in any special format? Or just sengind a string should work?

logger.info('hello world')

Thanks!

**Comment by klynch** [comment]
Hi, is the logger filtering out info statements by any chance? Did you try sending a message with warning or error priority?

**Comment by patrick-bark** [comment]
Yeah I am having the same issue. Does this support python 3? I have it set up the same as the example, but when I do a simple `logger.info('this is info')` I get:
`TypeError: a bytes-like object is required, not 'str'`
Any ideas here? Thanks

**Comment by patrick-bark** [comment]
Nevermind, it was a python3 thing, wrapping the pickle function in `bytes(self.format(record) + "\n", encoding='UTF-8')` worked good for me. A little python 2 disclaimer would be nice though. Thanks
